docs/conf.py

```diff
@@ -10,45 +10,45 @@
 
 import os
 
-import sphinx.environment
+# import sphinx.environment
```

Comment:
what was the need for commenting import?

Reply:
it is not used. but because this is the configuration file of the documentation generation i didn't want it to remove, because it would be helpful if someone would change that configuration again.